Update the game over message (#2)
kmaneesh23/skills-review-pull-requests: README.md

</header>

<!--
  <<< Author notes: Step 5 >>>

>   Start this step by acknowledging the previous step.

Author: ```suggestion
  Start this step by acknowledging the previous step. Add a back button to navigate previous lessons.
```

</header>

<!--
  <<< Author notes: Step 4 >>>
  Start this step by acknowledging the previous step.
  Define terms and link to docs.github.com.
-->

## Step 4: Suggest changes

_Nice work reviewing that pull request :sparkles:_

Now that you have explored the different ways you can review a pull request it is time to learn how to use _suggest changes_.

**What is _suggest changes_?**: This feature enables you to recommend a change to a pull request that the author can commit with the push of a button.

### :keyboard: Activity: Suggest changes

1. On the pull request, click **Files changed**.
1. Find the `index.html` changes.

> 1. Hover your cursor next to the line numbers on the left side of the page.

Author: - not left side of the page, rather left side of the code lines :P
```suggestion
1. Hover your cursor next to the line numbers on the left side of the page.
```

> 1. Hover your cursor next to the line numbers on the left side of the page.

Author: ```suggestion
1. Hover your cursor next to the line numbers on the left side of the code lines.
```
